projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/ProjectIntent.java

```diff
@@ -39,12 +47,11 @@ public boolean equals(Object o) {
 
     ProjectIntent that = (ProjectIntent) o;
 
-    return projectTitle.equals(that.projectTitle);
+    return projectTitle.equals(that.projectTitle) && projectObjective.equals(that.projectObjective);
   }
 
   @Override
   public int hashCode() {
-    return projectTitle.hashCode();
+    return projectTitle.hashCode(); //todo integrate objective hash -> du it with project intent?
```

Comment:
Maybe @KochTobi can tell me on how to calculate the hashcode for multiple values?

Reply:
Ask IntelliJ.
Simply press command + N and choose generate equals and hash code